src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -2,58 +2,55 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+#include "graph.h"
+
 #include <algorithm>
-#include <string>
+#include <fstream>
+#include <limits>
 #include <map>
-#include <vector>
+#include <memory>
+#include <string>
 #include <tuple>
-#include <unordered_set>
-#include <limits>
-#include <fstream>
 #include <unordered_map>
-#include <memory>
+#include <unordered_set>
 #include <utility>
+#include <vector>
 
-#include "graph.h"
-#include "graph_dumper.h"
-#include "graph_optimizer.h"
+#include "blob_factory.hpp"
+#include "common/primitive_desc.hpp"
+#include "common/primitive_desc_iface.hpp"
 #include "dnnl_extension_utils.h"
 #include "extension_mngr.h"
-#include "memory_solver.hpp"
-#include "itt.h"
+#include "graph_dumper.h"
+#include "graph_optimizer.h"
+#include "ie_algorithm.hpp"
+#include "ie_plugin_config.hpp"
 #include "infer_request.h"
-#include "nodes/input.h"
-#include <nodes/reorder.h>
+#include "itt.h"
+#include "low_precision/low_precision.hpp"
+#include "memory_desc/cpu_memory_desc_utils.h"
+#include "memory_desc/dnnl_blocked_memory_desc.h"
+#include "memory_solver.hpp"
+#include "ngraph/function.hpp"
```

Comment:
Please replace `ngraph` with `openvino`.

Reply:
sure, there are lots of ngraph calling in cpu plugin, we need additional PR to solve all of them. For this PR, it will solve this  issue only in the modified files.